src/metabase/analytics/prometheus.clj

```diff
@@ -295,6 +298,18 @@
      (setup!))
    (prometheus/inc (:registry system) metric labels amount)))
 
+(defn set!
```

Comment:
We could pick other name to avoid the clash, but it won't matter for namespaced references.